Rubberduck.RibbonDispatcher/Sample Ribbons/SampleRibbon/src/RibbonLoader.bas

```diff
@@ -0,0 +1,10 @@
+Attribute VB_Name = "RibbonLoader"
+Option Explicit
+
+Public Sub OnRibbonLoad(ByVal RibbonUI As Office.IRibbonUI)
+    On Error GoTo EH
+    ThisWorkbook.InitializeRibbonViewModel RibbonUI
+XT: Exit Sub
+EH: DisplayError Err, "RibbonLoader.OnRibbonLoad"
+    Resume XT
```

Comment:
Ok so this is the top of the call stack. This handler is needed then: rename labels, add a line break after them, and inline the `MsgBox` call.

Reply:
Really! DisplayError knows how to display the stack trace, and saves a lot of DRY code.

Reply:
I don't care for the "stack trace", and this is nothing more than a `MsgBox` wrapper method.